website/docs/r/alert_channel_aws_s3.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+subcategory: "Alert Channels"
+layout: "lacework"
+page_title: "Lacework: lacework_alert_channel_aws_s3"
+description: |-
+  Create and manage AWS S3 Alert Channel integrations
+---
+
+# lacework\_alert\_channel\_aws\_s3
+
+Create the S3 alert channel that Lacework uses to send the data to your designated bucket in AWS
+
+## Example Usage
+
+```hcl
+resource "lacework_alert_channel_aws_s3" "account_abc" {
+  name = "s3 Alerts"
+  credentials {
+    role_arn    = "arn:aws:iam::1234567890:role/lacework_iam_example_role"
+    bucket_arn  = "arn:aws:s3:::bucket_name/key_name"
+    external_id = "12345"
+  }
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The Alert Channel integration name.
+* `credentials` - (Required) The credentials needed by the integration. See [Credentials](#credentials) below for details.
+* `enabled` - (Optional) The state of the external integration. Defaults to `true`.
+
+### Credentials
+
+`credentials` supports the following arguments:
+
+* `role_arn` - (Required) The ARN of the IAM role.
+* `bucket_arn` - (Required) The ARN of the S3 bucket.
```

Comment:
Update the docs so that this field is not at the top level and not inside credentials